app.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+(function(){
+  angular.module('toDoApp', []).controller('list_controller', ListController);
+  function ListController(){
+    var toggleStatus = 0;
+    var vm = this;
+    vm.tasks = tasks;
+    vm.taskAdder = function(){
+      var taskBox = document.getElementById('task-field').value;
```

Comment:
instead of writing out vanilla JS to grab the input values, you could use the Angular directive `data-ng-model` instead